pages/clubs-toolkit.jsx

```diff
@@ -26,8 +28,6 @@ var Toolkit = React.createClass({
   render: function() {
     return (
       <div>
-        <p>{this.props.header}</p>
```

Comment:
I think these are not needed.  Especially for headers that have links within it will overcomplicate things (eg. [this header](https://github.com/mozilla/teach.webmaker.org/pull/683/files#diff-e1e44be96a2489f685e96b1a0240cfd8R123)).  We can turn header into a React component if we want to tidy things up.